Prediction_Eval.ipynb

```diff
@@ -0,0 +1,464 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
```

Comment:
<img width="668" alt="image" src="https://github.com/geo-smart/swe-workflow-book/assets/45752727/8e3ba0c3-b45b-4a4c-aa21-b4f4ae3ca944">

again, please dont use snake case for headings.